sale_outplacement/models/outplacement.py

```diff
@@ -108,7 +105,8 @@ def _get_department_id(self, data):
         department = self.env['performing.operation'].search(
             [('ka_nr', '=', data.get('utforande_verksamhet_id', ''))],
             limit=1)
-        _logger.debug('Department: hr_department %s | %s' % (department, data.get('utforande_verksamhet_id')))
+        _logger.debug('Department: hr_department %s | %s' % (
+            department, data.get('utforande_verksamhet_id')))
```

Comment:
Do we need this logging?

Reply:
Maybe maybe not, its a debug logg so it does not cost anything unless debugging is set